akd/src/tree_node.rs

```diff
@@ -800,6 +799,14 @@ impl TreeNode {
 
 /////// Helpers //////
 
+#[derive(Debug, Clone, Copy)]
+pub(crate) enum HashMode {
```

Comment:
naming nits:
- `HashMode` -> `LeafHashingMode`
- `WithLeafEpoch` -> `WithEpoch`
- `NoLeafEpoch` -> `WithoutEpoch`

Reply:
Technically the hashes that are affected are the parents of leaf nodes, not the leaf nodes themselves. Would it be confusing to call it `LeafHashingMode`?

Reply:
Ah, fair enough! How about `NodeHashingMode` in that case? I basically wanted to stay away from `HashMode` because it sounds a bit too generic and easily-confusable with some other hashing-related stuff. :)